pkg/uefi/file.go

```diff
@@ -218,9 +218,10 @@ type FileHeaderExtended struct {
 
 // File represents an EFI File.
 type File struct {
-	Header   FileHeaderExtended
-	Type     string
-	Sections []*Section `json:",omitempty"`
+	Header    FileHeaderExtended
+	Type      string
+	Sections  []*Section `json:",omitempty"`
```

Comment:
Can you put a comment in front of this saying files either have `Sections` or `NVarStore`, but not both?

Reply:
done